plugins/rms_quotes/rms_quotes.rb

```diff
@@ -0,0 +1,33 @@
+#  -*- coding: utf-8 -*-
+#  rms_quotes.rb
+#  Author: Winston Weinert
+#  ------------------------
+#  A Cinch plugin that provides Richard Stallman (rms) quotes for yossarian-bot.
+#  rms_quotes.txt is parsed from https://github.com/faiq/rms
+#  ------------------------
+#  This code is licensed by Winston Weinert under the MIT License.
+#  http://opensource.org/licenses/MIT
+
+require_relative '../yossarian_plugin'
+
+class RmsQuotes < YossarianPlugin
```

Comment:
Maybe `RMSQuotes` for uniformity with other plugins? See `CTCPVersion`, `IPInfo`, etc.